asset-transfer-basic/application-javascript/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "asset-transfer-basic",
+    "version": "1.0.0",
+    "description": "Asset-transfer-basic application implemented in JavaScript",
+    "engines": {
+        "node": ">=8",
```

Comment:
would push this up to match the latest LTS for node.. (applies to all the other package.json).

Reply:
done.